validator/validator-main.protoascii

```diff
@@ -3303,6 +3303,14 @@ attr_lists: {
   }
 }
 
+# Video Interface attributes. These attributes can be attached to any
+# element that implements the video interface
+attr_lists: {
+  name: "implements-video-interface"
+  attrs: { name: "autoplay" }
+  attrs: { name: "dock" }
```

Comment:
@aghassemi Only `amp-video` is for both `AMP` and `AMP4ADS`. We can keep it in `attr_list` but we'll need to create two separate tagspecs for `amp-video`, one for `html_format: AMP` and one for `html_format: AMP4ADS`. Only `html_format: AMP` would have the `attr_list` but otherwise they'd look identical.